packages/schematics/angular/application/files/karma.conf.js.template

```diff
@@ -13,6 +13,11 @@ module.exports = function (config) {
       require('@angular-devkit/build-angular/plugins/karma')
     ],
     client: {
+      jasmine: {
+        // if you run into a randomly failing test, you can find the culprit seed in the browser window when running `ng test`
+        // you can then run the same test order with `JASMINE_SEED=123 ng test`
```

Comment:
I don't think this command works cross-platform. 

In general, I am not sure the added benefit of this.  I don't think that when a test fails and user figure that it's because of randomization they will go and look `karma.conf.js`. I find environment variables are not that discoverable.

//cc @clydin what do you think?

Reply:
No worries, feel free to discard the PR.
I've seen a lot of teams struggling with this issue and it can be hard to find how to set the seed, so I thought it might be worth it :)

Reply:
What about if we leave the `jasmine` object (but empty) and add a comment explaining that jasmine specific options can be placed here?  A link to the available jasmine options may be useful as well.   If there are commonly used options, they could potentially also be listed with a brief description on when they would be useful.

Reply:
Agreed. that would be a better approach!

Reply:
Sounds good. I pushed a new commit if you want to take another look.
I also updated the commit and PR title/description.

Reply:
@alan-agius4 I just updated the config template, but I can of course update the existing karma.conf in the project as I did for #19356 once we agree on the content

Reply:
No need to add this change there. Thanks